classport-agent/src/test/java/ClassportAgentTest.java

```diff
@@ -0,0 +1,164 @@
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.junit.jupiter.api.Test;
+
+import io.github.chains_project.classport.agent.ClassportAgent;
+import io.github.chains_project.classport.commons.AnnotationReader;
+import io.github.chains_project.classport.commons.ClassportInfo;
+
+public class ClassportAgentTest {
+
+    private static final String RESOURCES_PATH = "src/test/resources/";
+    private static final String CLASSPORT_TREE_PATH = RESOURCES_PATH + "classport-deps/classport-deps-tree";
+    private static final String CLASSPORT_LIST_PATH = RESOURCES_PATH + "classport-deps/classport-deps-list";
+    private static final String ANNOTATED_CLASS_PATH = RESOURCES_PATH + "annotated-classes/StringUtils.class";
+    private static final String NOT_ANNOTATED_CLASS_PATH = RESOURCES_PATH + "not-annotated-classes/StringUtils.class";
+
+    @Test
+    void shouldGenerateDependencyListAndTreeFiles() throws Exception {
```

Comment:
I assumed this test is about test [this method](https://github.com/chains-project/classport/blob/7df77a244d74341d7392a3a29ee834cd23191610/classport-analyser/src/main/java/io/github/chains_project/classport/analyser/Analyser.java#L33) but you have created another `getSBOM` which returns mock values. The test should really assert if the methods in main method are working correctly.